src/App.css

```diff
@@ -434,4 +434,88 @@ input {
     --search-box-h: 3rem !important;
     --search-box-w: 90% !important;
   }
+
+  .col {
+    width: 25%;
+    grid-template-columns: auto;
+    grid-gap: 20px;
```

Comment:
Try using `grid-template-rows` as well, and also in the media queries. The whole point is to have every `.col` in its own row on smaller devices.
![respo2](https://user-images.githubusercontent.com/60689731/82793192-2e1b6480-9e79-11ea-82ca-e51f82607cbc.png) The footer still looks the same on smaller devices.

Reply:
okay still working on it but it should be looking like this right now from the most recent push 
![small screeen screenshot](https://user-images.githubusercontent.com/45368398/82840411-c3971280-9ed2-11ea-95fb-0b3cbc292946.png)